website/docs/gitopssets/api-reference.md

```diff
@@ -0,0 +1,10 @@
+---
+title: API reference
+hide_title: true
+---
+
+import GeneratedAPI from './_api.md';
```

Comment:
if we make this `api-reference.mdx` instead it should avoid adding in these funny `/` escapes